### ISSUE TITLE
Operating System 9.2 and RF-Link

### ISSUE THREAD

**Issue by RGN01** [home-assistant/operating-system#2184]
### Describe the issue you are experiencing

Is there a known issue with RF-Link and Home Assistant 2022.10.3, Supervisor 2022.10.0, Operating System 9.2?

I updated to OS 9.2 earlier this evening and now RF-Link no longer works. Errors in logs recurring at every attempt to use RF-Link and something like this error during decode of data, invalid data: ��a�x�� (hieroglyphics appear exactly like that in the logs).

Hardware is a NUC, RF-Link is attached via USB

### What operating system image do you use?

generic-x86-64 (Generic UEFI capable x86-64 systems)

### What version of Home Assistant Operating System is installed?

9.2

### Did you upgrade the Operating System.

Yes

### Steps to reproduce the issue

1. I upgraded to OS 9.2 and rebooted
2. RF-Link no longer worked
3. errors in logs repeatedly and when trying to run any RF-Link command
4. Have tried further reboots but that didn't work.

I'm out of my depth and don't know what else to try so any help would be appreciated. Thanks.

### Anything in the Supervisor logs that might be useful for us?

```txt
Logger: rflink.protocol
Source: runner.py:119
First occurred: 18:08:12 (111 occurrences)
Last logged: 20:14:43

Error during decode of data, invalid data: ��,��
Error during decode of data, invalid data: ��&&��e ���7 ��=��
Error during decode of data, invalid data: �l����
Error during decode of data, invalid data: ���I��
Error during decode of data, invalid data: ��+��


And:

Logger: homeassistant.components.websocket_api.http.connection
Source: components/rflink/__init__.py:562
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 18:11:04 (4 occurrences)
Last logged: 19:37:43

[139742588150896]
[139742178449472]
[139742176920816]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/asyncio/locks.py", line 214, in wait
    await fut
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/asyncio/tasks.py", line 456, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 202, in handle_call_service
    await hass.services.async_call(
  File "/usr/src/homeassistant/homeassistant/core.py", line 1738, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1775, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 207, in handle_service
    await service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 678, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 931, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 715, in _handle_entity_call
    await result
  File "/usr/src/homeassistant/homeassistant/components/rflink/cover.py", line 156, in async_close_cover
    await self._async_handle_command("close_cover")
  File "/usr/src/homeassistant/homeassistant/components/rflink/__init__.py", line 536, in _async_handle_command
    await self._async_send_command(cmd, self._signal_repetitions)
  File "/usr/src/homeassistant/homeassistant/components/rflink/__init__.py", line 562, in _async_send_command
    await self._protocol.send_command_ack(self._device_id, cmd)
  File "/usr/local/lib/python3.10/site-packages/rflink/protocol.py", line 257, in send_command_ack
    yield from asyncio.wait_for(self._command_ack.wait(), TIMEOUT.seconds)
  File "/usr/local/lib/python3.10/asyncio/tasks.py", line 458, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

And:

Logger: homeassistant.components.automation.lounge_blind_close_at_sunset
Source: components/rflink/__init__.py:562
Integration: Automation (documentation, issues)
First occurred: 18:14:22 (2 occurrences)
Last logged: 18:14:22

Lounge Blind - close at Sunset: Error executing script. Unexpected error for call_service at pos 2:
While executing automation automation.lounge_blind_close_at_sunset
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/asyncio/locks.py", line 214, in wait
    await fut
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/asyncio/tasks.py", line 456, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 451, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 684, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1738, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1775, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 207, in handle_service
    await service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 678, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 931, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 715, in _handle_entity_call
    await result
  File "/usr/src/homeassistant/homeassistant/components/rflink/cover.py", line 156, in async_close_cover
    await self._async_handle_command("close_cover")
  File "/usr/src/homeassistant/homeassistant/components/rflink/__init__.py", line 536, in _async_handle_command
    await self._async_send_command(cmd, self._signal_repetitions)
  File "/usr/src/homeassistant/homeassistant/components/rflink/__init__.py", line 562, in _async_send_command
    await self._protocol.send_command_ack(self._device_id, cmd)
  File "/usr/local/lib/python3.10/site-packages/rflink/protocol.py", line 257, in send_command_ack
    yield from asyncio.wait_for(self._command_ack.wait(), TIMEOUT.seconds)
  File "/usr/local/lib/python3.10/asyncio/tasks.py", line 458, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError
```

### Anything in the Host logs that might be useful for us?

```txt
Nothing with the RF-link text
```

### System Health information

Home Assistant 2022.10.3
Supervisor 2022.10.0
Operating System 9.2
Frontend 20221010.0 - latest

I cannot see anything else

**Comment by RGN01** [comment]
I reverted to OS 9.1 and it was still problematic. 

I then discovered that the USB port assigned to the RF-Link had changed from ttyACM0 to ttyACM1. I switched it back and it worked again.

Note that the setting is a manual entry in configuration.yaml and I did not edit it, nor did I or anyone else go near the HASS server (it is in a locked cabinet).

I have again upgraded to 9.2 and it is working. USB port did not change this time.

It's all very strange but it seems OK now.
